tiny_gltf.h

```diff
@@ -2804,6 +2833,51 @@ static bool ParseBufferView(BufferView *bufferView, std::string *err,
   bufferView->byteOffset = static_cast<size_t>(byteOffset);
   bufferView->byteLength = static_cast<size_t>(byteLength);
   bufferView->byteStride = static_cast<size_t>(byteStride);
+  return true;
+}
+
+static bool ParseSparseAccessor(Accessor *accessor, std::string *err,
```

Comment:
Here's the function that actually parse the "sparse" json object. It will be called by ParseAccessor only if it is needed.